contents/blog/posthog-vs-launchdarkly.md

```diff
@@ -0,0 +1,566 @@
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Calculate the recommended run time for your experiments</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+PostHog automatically calculates a recommended run time based on past data and minimally acceptable improvements. This helps you avoid the peeking problem and end your experiment at the right time.
+
+![Run time calculator](../images/blog/posthog-vs-launchdarkly/runtime.png)
+
+### Pricing
+
+PostHog and LaunchDarkly price significantly differently. 
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td className="text-center">PostHog</td>
+            <td className="text-center">LaunchDarkly</td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Pricing</td>
+            <td className="text-center">Simple per request pricing</td>
+            <td className="text-center">Based on MAU + seats + add-ons</td>
+        </tr>
+        <tr>
+            <td>Free plan</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center">Trial only</td>
+        </tr>
+        <tr>
+            <td>Free experiments</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        </tr>
+        <tr>
+            <td>Free collaboration</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+PostHog’s feature flag pricing is pay-per-request (and A/B tests use feature flags). There is a generous free tier of 1M requests per month with all features, add-ons, and integrations available.
+
+![PostHog pricing](../images/blog/posthog-vs-launchdarkly/posthog-price.png)
+
+LaunchDarkly’s pricing is more complicated. They charge by seat starting at $10/seat/month. On top of this, they charge based on "monthly context instances" (MCIs) also known as monthly active users. The pro plan only gets 1,000 frontend MCIs per month.
+
+![LaunchDarkly pricing](../images/blog/posthog-vs-launchdarkly/launchdarkly-price.png)
+
+As mentioned earlier, many of the security, approval, and workflow features that differentiate them are only available at the opaque "Enterprise" level. 
+
+### Reporting
+
+LaunchDarkly features basic reporting but requires other platforms for more advanced analysis. PostHog builds in advanced analysis like SQL querying, conversion funnels, and dashboards and combines flag data with other product data.
```

Comment:
May be worth expanding this out to be more explicit and to clarify the difference between 'reporting' and 'analytics'. 

The first sentence also reads like it requires (and has) integrations. May be better to just say it can't do more advanced analysis.